app/javascript/registration/register.js

```diff
@@ -26,6 +26,14 @@ export default function DorRegistration(initOpts) {
       return document.querySelector('[data-rcparam="workflowId"]').value
     },
 
+    contentType : function() {
+      return document.querySelector('[data-rcparam="contentType"]').value
+    },
+
+    viewingDirection : function() {
+      return document.querySelector('[data-rcparam="viewingDirection"]').value
```

Comment:
Will this always return a viewingDirection since it's not checking to see if the form element is disabled?

Reply:
It will, but it's getting dropped downstream when creating the cocina model unless it's a book or image tyoe.  I'll change this function to return null unless the element is enabled though so we don't even send it.